daemon/graphdriver/overlay2/overlay.go

```diff
@@ -174,6 +175,14 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 		}
 	}
 
+	// If kernel is older than 4.8, check whether opaque copy up bug has been patched
+	if kernel.CompareKernelVersion(*v, kernel.VersionInfo{Kernel: 4, Major: 8, Minor: 0}) < 0 {
+		if err := hasOpaqueCopyUpBug(home); err != nil {
+			logrus.Warnf("Not using native diff for overlay2: %v", err)
```

Comment:
How about printing something like "Please consider updating the kernel to 4.8.0 or later" here